code/game/gamemodes/machinegod/machinegod.dm

```diff
@@ -0,0 +1,30 @@
+
+
+/datum/game_mode/machinegod
```

Comment:
Would it not be easier to inherit from cult?
You get a lot of the setup code, surely, and I notice a few copied things (harvest_target).